doc/chapters/api/index.rst

```diff
@@ -24,88 +13,276 @@ Methods
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        - **None**
+
+:Return Value:
+        - **None** 
+
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
```

Comment:
We have used some of similar examples at Getting started chapter. So i just want to use same schema. But if it would be beteer we can create a small section which explain very basics of dbus-send. Or it is possible to give reference to its web page.